SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -33,11 +33,11 @@ public ProcessDataGateway(ISccvDbContext sccvDbContext, DatabaseContext database
             FilterDefinition<BsonDocument> firstNameFilter;
             FilterDefinition<BsonDocument> lastNameFilter;
 
-            if (!string.IsNullOrWhiteSpace(request.MosaicId))
+            if (!string.IsNullOrWhiteSpace(request.RecordId))
             {
-                var query = _sccvDbContext.getCollection().AsQueryable();
```

Comment:
Please leave this in place for when MosaicId is used

Reply:
Added back in